src/DotNetWorker.Grpc/Definition/GrpcFunctionDefinition.cs

```diff
@@ -19,9 +19,21 @@ public GrpcFunctionDefinition(FunctionLoadRequest loadRequest, IMethodInfoLocato
         {
             EntryPoint = loadRequest.Metadata.EntryPoint;
             Name = loadRequest.Metadata.Name;
-            PathToAssembly = Path.GetFullPath(loadRequest.Metadata.ScriptFile);
             Id = loadRequest.FunctionId;
 
+            string? scriptRoot = Environment.GetEnvironmentVariable("AzureWebJobsScriptRoot");
+            if (string.IsNullOrWhiteSpace(scriptRoot))
```

Comment:
These checks and throw were added to figure out why the process was crashing, while we were building the first working version. Should we keep this?

Reply:
Discussed offline. Keeping checks and updating verbiage.